### ISSUE TITLE
Provide warnings/errors during conversions

### ISSUE THREAD

**Issue by jonboulle** [appc/docker2aci#24]
As noted in https://github.com/appc/docker2aci/pull/20#discussion_r24527266 , it would be good to be able to warn people about cases when docker2aci has to perform suboptimal translations (for example when converting an exec statement to use /bin/sh)

**Comment by jonboulle** [comment]
@iaguis since we started down this path with #43 perhaps we can add this more widely now?

**Comment by dghubble** [comment]
Agreed. I investigated several red herrings before realizing `ENTRYPOINT ["./binary"]` vs `ENTRYPOINT ["/binary"]` was the root issue causing problems due to arguments being ignored in a fetched image. A log line would have saved time.
https://github.com/coreos/coreos-baremetal/pull/39

**Comment by jonboulle** [comment]
Hmm although for your particular issue, since absolute paths are no longer a requirement in the spec, maybe we should relax this conversion now? @iaguis thoughts?

**Comment by iaguis** [comment]
Makes sense: https://github.com/appc/docker2aci/pull/114
